lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -180,6 +246,28 @@ class ConvertAtenSigmoidOp : public OpConversionPattern<AtenSigmoidOp> {
   }
 };
 
+class ConvertAtenSqrtOp : public OpConversionPattern<AtenSqrtOp> {
+public:
+  using OpConversionPattern<AtenSqrtOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenSqrtOp::Adaptor;
+
+  LogicalResult
+  matchAndRewrite(AtenSqrtOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    Value input = adaptor.getSelf();
+    RankedTensorType inputType = input.getType().dyn_cast<RankedTensorType>();
+    if (!inputType)
+      return rewriter.notifyMatchFailure(
+          op, "Only Ranked Tensor types are supported in TCP");
+    if (!inputType.getElementType().isa<mlir::FloatType>())
+      return rewriter.notifyMatchFailure(
+          op, "Sqrt input tensor must have floating-point datatype");
```

Comment:
We are repeating this in every op now. We should probably extract this out into a helper function.

Not for this PR, probably do it as a separate cleanup PR.

Reply:
Sounds good. Let's do that in a separate PR.